app/models/hud_reports/report_client_base.rb

```diff
@@ -72,7 +72,7 @@ def fetch_cell_value(col)
     def format_complex_value(col, value, pii_policy, include_content_tag)
       if value.is_a?(Array)
         # For Arrays, calculate each array element's value using the column name for the array
-        value.map { |item| display_value(col, pii_policy: pii_policy, include_content_tag: include_content_tag, cell_val: item, calculate_cell: false) }
+        value = value.map { |item| display_value(col, pii_policy: pii_policy, include_content_tag: include_content_tag, cell_val: item, calculate_cell: false) }
```

Comment:
This appears to be a broader bug, causes values containing arrays to skip display value

Note, the report client base seems like kind of a lot so I opted to build a presenter for the hopwa caper instead